warehouse/templates/manage/settings.html

```diff
@@ -19,7 +19,8 @@
 
 {% block main %}
   <h2>{% trans %}Project settings{% endtrans %}</h2>
-
+  {% trans  %}Project size :{% endtrans %}
+  {{ project.total_size|filesizeformat(binary=True) }}<br><br>
```

Comment:
```suggestion
  {{ project.total_size|filesizeformat(binary=True) }}
```
I think it might make sense to put all of these in a `<ul>` tag instead of trying to manage whitespace with line breaks here.

Reply:
I think my new changes will work this out